Controller/Admin/TransactionTabData.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Core\Helper;
+use Wirecard\Oxid\Controller\Admin\TransactionTab;
+
+/**
+ * Controls the view for the data transaction tab.
+ */
+class TransactionTabData extends TransactionTab
```

Comment:
Please look at pg_response_mapper.php in opencart-ee.php there you will find an example how we solved the getting data from response.